serverSettings.js

```diff
@@ -0,0 +1,22 @@
+var settings = {
```

Comment:
Please document this module. It is not clear what each method does unless you dig in the each file to find a reference to it.